hc-venv/bin/activate

```diff
@@ -0,0 +1,78 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
Why are your env files in the repo?

Reply:
Issue has been resolved, there was an issue with the naming of my env in the gitignore file